qt/applications/workbench/workbench/plotting/test/test_figuremanager.py

```diff
@@ -1,4 +1,16 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
 import unittest
+
+from qtpy.QtWidgets import QApplication
+
+from mantidqt.widgets.workspacedisplay.test_helper.qt_widget_finder import QtWidgetFinder
```

Comment:
Would this be better in the `mantidqt.utils.qt.test` package as I think it's more general?

Reply:
Good idea